.github/workflows/bump-golang.yml

```
@@ -56,7 +32,22 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ steps.get_token.outputs.token }}
           GITHUB_BRANCH: 'main'
-          GITHUB_LABELS: ${{ needs.labels.outputs.backports }}
+          GITHUB_LABELS: 'backport-active-9'
+
+  bump-819:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          ref: '8.19'
+
+      - uses: elastic/oblt-actions/updatecli/run@v1
+        with:
+          command: --experimental apply --config .ci/updatecli/bump-golang.yml --values .ci/updatecli/values.d/scm.yml
```

`nit`: no need for experimental anymore

```suggestion
          command: apply --config .ci/updatecli/bump-golang.yml --values .ci/updatecli/values.d/scm.yml
```